README.md

```diff
@@ -3,11 +3,38 @@
 Ventana is a client to help connecting and making requests to a SugarCRM REST
 API on multiple versions.
 
-Currently this library provides only a Javascript connector. Please contribute
-to help us grow this client connector.
+Currently this library provides a JavaScript connector.
+
+Alertnatively, a PHP connector is available {@link https://github.com/sugarcrm/rest-php-client here}
```

Comment:
"Alertnatively" => "Alternatively"

Also, if you use GitHub link syntax rather than JSDoc link syntax, the link will render properly when read both on GH and on JSDoc.